contrib/omnihub/src/omnihub/frameworks/pytorch.py

```diff
@@ -0,0 +1,236 @@
+import os
+from omnihub.model_hub import omnihub_dir, ModelHub
+import torch
+from  torchvision import models
+import numpy as np
+
+framework_name = 'pytorch'
+framework_dir = os.path.join(omnihub_dir, framework_name)
+BASE_URL = 'https://s3.amazonaws.com/pytorch/models'
+
+
+class PytorchModelHub(ModelHub):
+    def __init__(self):
+        super().__init__(framework_name,BASE_URL)
+
+
+    def download_model(self, model_path,**kwargs) -> str:
+        model = None
+        if model_path == 'resnet18':
+            height = kwargs.get('height',224)
+            width = kwargs.get('width',224)
+            x = torch.from_numpy(np.ones((1, 3, height, width),dtype=np.float32))
+            model = models.resnet18(pretrained=True,**kwargs)
```

Comment:
This looks very much like you've hardcoded something that should have been a `dict`. 

In particular, I think you can probably put the default input shape into a dict indexed by the modelname, and use `getattr` on `models` to reference the actual model function.

Or have a dict that holds on to both input shape and model generation with two lambdas.

Reply:
@treo ok broke this up a bit and also made the model name generation a bit better. It's still a bit messy but at least reduces the boilerplate a bit.